src/jit/bitsetasshortlong.h

```diff
@@ -664,9 +664,11 @@ void BitSetOps</*BitSetType*/ BitSetShortLongRep,
                /*BitSetTraits*/ BitSetTraits>::ClearDLong(Env env, BitSetShortLongRep& bs)
 {
     assert(!IsShort(env));
-    // Recall that ClearD does *not* require "bs" to be of the current epoch.
-    // Therefore, we must allocate a new representation.
-    bs = MakeEmptyArrayBits(env);
```

Comment:
Possibly, but `CheckEpoch` is an instance method on class `BitSetUint64` that asserts that the `m_epoch` field of that class matches the provided epoch (i.e. the epoch retrieved from `Env`). The short/long bitset implementation does not have an equivalent of `m_epoch` that it can use to perform this check (at least not that I can find, but perhaps I'm missing something).